src/stores/models/Vote.js

```diff
@@ -0,0 +1,42 @@
+import { TransactionStatus } from 'constants';
+
+import { satoshiToDecimal, decimalToSatoshi } from '../../helpers/utility';
+
+/**
+ * Model for Transactions.
+ */
+export default class Vote {
+  type = '' // One of: DBVoteType
+  txid = '' // Transaction ID assigned by the blockchain
+  status = '' // only success
+  block
+  blockNum = 0 // Block number when Transaction was recorded on blockchain
+  blockTime = '' // Block timestamp for blockNum
+  voterAddress = '' // Sender's address
+  topicAddress = '' // Topic contract address associated with Transaction
+  oracleAddress = '' // Oracle contract address associated with Transaction
+  optionIdx = 0 // Result index used for Transaction. eg. For a bet, this would be the result index the user bet on.
+  amount = '' // Token amount in decimals
+  amountSatoshi = '' // Token amount in satoshi
+  token = '' // Token type used for Transaction. QTUM for BET. BOT for VOTE.
+  version = 0 // Current version of the contract. To manage deprecations later.
+  localizedInvalid = {}; // for invalid option
+  topic
```

Comment:
lets remove `topic` and `localizedInvalid` from the Vote model. and you can pull that from the Topic obj in the Event store.